src/public/style.css

```diff
@@ -1,12 +1,72 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather|Open+Sans');
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Open Sans', sans-serif;
+}
+
 body {
-  background-color: #d9e6fc;
+  font-family: 'Merriweather', serif;
+}
+
+
+
+header {
+  margin: 0 !important;
+  display: block;
+}
+
+div.page {
+    background-color: lightgrey;
+    }
+
+div.title {
+  position: static;
+}
+
+div.content-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: flex-start;
+  align-content: flex-start;
+  }
+
+main {
+  padding: 20px 20px 20px 20px;
 }
-h1 {
-  font-size: 58px;
-  background-color: #21e342;
+
+nav {
+  padding: 20px 20px 20px 20px;
+}
+
+div.nav-width {
+    align-items: flex-start;
+    flex: 4;
+}
+
+div.main-width {
+  align-items: flex-start;
+  flex: 8;
+}
+
+footer {
+  display: block;
+  text-align: center;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.plaintext-color {
+  color: #4a4a4a;
```

Comment:
While this works, it might be better to simply target `body` to change the color of all text in the document. This way you don't need to apply the class everywhere.